src/dataflow/src/render/join.rs

```diff
@@ -334,19 +335,34 @@ where
         Tr2::Batch: BatchReader<Row, Tr2::Val, G::Timestamp, repr::Diff> + 'static,
         Tr2::Cursor: Cursor<Row, Tr2::Val, G::Timestamp, repr::Diff> + 'static,
     {
-        let mut row_packer = repr::RowPacker::new();
-        prev_keyed.join_core(&next_input, move |_keys, old, new| {
-            let prev_datums = old.unpack();
-            let next_datums = new.unpack();
-            // TODO: We could in principle apply some predicates here, and avoid
-            // constructing output rows that will be filtered out soon.
-            Some(
-                row_packer.pack(
-                    prev_datums
-                        .iter()
-                        .chain(next_vals.iter().map(|i| &next_datums[*i])),
-                ),
-            )
-        })
+        use differential_dataflow::AsCollection;
+        use timely::dataflow::operators::OkErr;
+
+        // Reuseable allocation for unpacking.
+        let mut datums = DatumVec::new();
+        let mut row_packer = RowPacker::new();
+        // let mut row_packer = repr::RowPacker::new();
```

Comment:
this seems like a stray commented out line

Reply:
Yup. Fixed.